### PR TITLE
[WIP] Fix tor version 3 support for networking

### PR DESCRIPTION
**Pull request by marpme**
Fix the netaddress conversation between other nodes for tor-v3-nodes.

### CODE REVIEW

File: src/netaddress.cpp [hunk 1]
```diff
@@ -15,6 +15,7 @@ static const unsigned char pchGarliCat[] = {0xFD,0x60,0xDB,0x4D,0xDD,0xB5};
 
 // 0xFD + sha256("bitcoin")[0:5]
 static const unsigned char g_internal_prefix[] = { 0xFD, 0x6B, 0x88, 0xC0, 0x87, 0x24 };
+static const unsigned char pchOnionv3Cat[] = { 0xFD, 0x50, 0xa7, 0xf7, 0xc0, 0xf8};
```
Comment (marpme) [inline src/netaddress.cpp]: append comment how it was built

File: src/netaddress.h [hunk 2]
```diff
@@ -10,6 +10,7 @@
 #include <config/verge-config.h>
 #endif
 
+#include <logging.h>
```
Comment (marpme) [inline src/netaddress.h]: Unused remove

File: src/netaddress.h [hunk 3]
```diff
@@ -75,7 +75,7 @@ class CNetAddr
         bool IsRFC6052() const; // IPv6 well-known prefix (64:FF9B::/96)
         bool IsRFC6145() const; // IPv6 IPv4-translated address (::FFFF:0:0:0/96)
         bool IsTor() const;
-		    bool IsI2P() const;
+		bool IsI2P() const;
```
Comment (marpme) [inline src/netaddress.h]: check indentation

**Comment by justinvforvendetta** [comment]
tested, works great. not sure what circles problem is, but we can see that after.

**Pull request merged**